wiki/Tournaments/ICC/2021/en.md

```diff
@@ -0,0 +1,277 @@
+| **H** | ![][flag_ID] **[LeWind](https://osu.ppy.sh/users/9718235) (7 pts)** | ![][flag_ID] **[Ruu](https://osu.ppy.sh/users/3212755) (11 pts)** | ![][flag_ID] [Francesca](https://osu.ppy.sh/users/8995235) (0 pts) | ![][flag_ID] [OrdinaryDaffa](https://osu.ppy.sh/users/8404301) (6 pts) | [#1](https://osu.ppy.sh/community/matches/90799873) |
+
+## Ruleset
+
+### General rules
+
+1. Map scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The mapsets for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. If there are any players who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#reschedule-request` channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a player does not show up within **10 minutes** of the start time, their opponent wins by default.
+6. If no staff or referee is available, the match will be postponed.
+7. If a player disconnects, it will be treated as if they had failed the beatmap.
+   - Disconnects that occur within a few seconds after the beatmap is started by the referee can be rematched.
+8. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most.
+   - In case the disconnected player failed to report back after the maximum allowance of 15 minutes had passed, the opposing player may be declared to be winning the match by default.
+9. Lag is not a valid reason to nullify a beatmap.
+10. If any problems during the match occur, the Tournament Management will make a decision based on referee's report.
+11. It is expected that all players be polite and respectful to each other. Penalties will be given if players violate.
+    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from the tournament altogether by the Tournament Management.
+    - Usage of any tools and programs that are against [osu!'s community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually.
+   - In order to be eligible to play in the tournament, each player is required to have either the Indonesian flag displayed on their profile and a playcount of 3,000 or more in osu!catch.
```

Comment:
either part of the sentence is missing, or the "either" should be removed

```suggestion
   - In order to be eligible to play in the tournament, each player is required to have the Indonesian flag displayed on their profile and a playcount of 3,000 or more in osu!catch.
```

Reply:
ya the `either` here was a mistake whoops